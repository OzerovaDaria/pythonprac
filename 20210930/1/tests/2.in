11,52,87,65,44
