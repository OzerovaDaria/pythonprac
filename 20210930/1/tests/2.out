[52, 11, 65, 44, 87]
